src/skvhttpclient/src/skvhttp/client/SKVClient.cpp

```diff
@@ -39,6 +39,15 @@ boost::future<Response<>> Client::createCollection(dto::CollectionMetadata metad
     return _HTTPClient.POST<dto::CollectionCreateRequest>("/api/CreateCollection", std::move(request));
 }
 
+boost::future<Response<>> Client::dropCollection(const String& collectionName) {
+    dto::CollectionDropRequest request{
+        .name = std::move(collectionName)
```

Comment:
A little surprised this compiles without a warning, but you should not move a const&. Pass by copy instead

Reply:
Done